src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -56,6 +58,8 @@ public partial class InfiniteScrollList : UserControl
 
         private const string LogEntrySource = "NuGet Package Manager";
 
+        private bool FilterByVulnerabilities = false;
```

Comment:
Violates naming for private instance variables

```suggestion
        private bool _filterByVulnerabilities = false;
```